core/modules/filters/x-listops.js

```diff
@@ -23,13 +23,22 @@ Extended filter operators to manipulate the current list.
         return results;
     };
 
+    /*
+    Return the number the str represents
+    Return default if it's not a number
+    */
+    function getInt( str, deflt ) {
+    	var i= parseInt( str );
```

Comment:
We should provide 10 as the radix parameter: `var i = parseInt(str);`

Also note that house style is that there should be a space on either side of the "=" of an assignment

Reply:
0 is not required as it is the default, isn't it?

Did it nonetheless.